KD_Lib/Pruning/lottery_tickets/lottery_tickets.py

```diff
@@ -97,6 +97,7 @@ def _original_initialization(self):
 
     def _train_after_pruning(self, prune_it):
         best_acc = 0.0
+        best_weights = None
```

Comment:
initialize this to model weights (the reason being if the accuracy doesn't improve then weights will be None)

Reply:
I have already included a condition while saving the model that if best_weights are None, then nothing needs to be done